pkg/etcdcli/interfaces.go

```diff
@@ -20,6 +26,8 @@ type EtcdClient interface {
 
 	GetMember(name string) (*etcdserverpb.Member, error)
 	MemberUpdatePeerURL(id uint64, peerURL []string) error
+
+	Refresh() error
```

Comment:
feels like an abstraction leak, maybe the impl should take care of this all by itself during construction or maybe the interface method could be `Init` or something which should be called before use?